lib/device.ex

```diff
@@ -78,9 +78,10 @@ defmodule Onvif.Device do
            device_from_probe_match(probe_match, username, password, prefer_https?, prefer_ipv6?),
          {:ok, device_with_datetime} <- get_date_time(device),
          {:ok, updated_device} <- guess_auth(device_with_datetime) do
-      updated_device
-      |> get_services()
-      |> set_media_service_path()
+      {:ok,
```

Comment:
did you want to do this for the `new` function too? or because it doesn't do any validating just leave it?

Reply:
Ya, we can leave the new function as it is.